# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1269460105 14400
#      Wed Mar 24 15:48:25 2010 -0400
# Node ID 91d1e64804e1c3c17e94931bb8fa78d8c4ed721f
# Parent  e85fa73ac3a03a3830a5d924bc8e8d760ca6c354
point to bugs.octave.org bug tracker instead of bug@octave.org mailing list

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-03-24  John W. Eaton  <jwe@octave.org>
+
+	* interpreter/bugs.txi: Update for bug tracker.
+
 2010-02-11  Jaroslav Hajek  <highegg@gmail.com>
 
 	* interpreter/expr.txi: Mention no ternary operator and the ifelse
 	fucntion as a possible replacement.
 
 2010-02-09  Jaroslav Hajek  <highegg@gmail.com>
 
 	* interpreter/tips.txi: More tips.
diff --git a/doc/interpreter/bugs.txi b/doc/interpreter/bugs.txi
--- a/doc/interpreter/bugs.txi
+++ b/doc/interpreter/bugs.txi
@@ -103,30 +103,19 @@ report the problem.
 Reporting a bug may help you by bringing a solution to your problem, or
 it may not.  In any case, the principal function of a bug report is
 to help the entire community by making the next version of Octave work
 better.  Bug reports are your contribution to the maintenance of Octave.
 
 In order for a bug report to serve its purpose, you must include the
 information that makes it possible to fix the bug.
 
-@findex bug_report
-
-If you have Octave working at all, the easiest way to prepare a complete
-bug report is to use the Octave function @code{bug_report}.  When you
-execute this function, Octave will prompt you for a subject and then
-invoke the editor on a file that already contains all the configuration
-information.  When you exit the editor, Octave will mail the bug report
-for you.
-
-@DOCSTRING(bug_report)
-
 @menu
 * Bug Criteria::                
-* Where: Bug Lists.             Where to send your bug report.
+* Where: Bug Tracker.           Where to submit your bug report.
 * Reporting: Bug Reporting.     How to report a bug effectively.
 * Patches: Sending Patches.     How to send a patch for Octave.
 @end menu
 
 @node Bug Criteria
 @appendixsec Have You Found a Bug?
 @cindex bug criteria
 
@@ -173,44 +162,34 @@ practice''.
 
 @cindex improving Octave
 @cindex suggestions
 @item
 If you are an experienced user of programs like Octave, your suggestions
 for improvement are welcome in any case.
 @end itemize
 
-@node Bug Lists
+@node Bug Tracker
 @appendixsec Where to Report Bugs
-@cindex bug report mailing lists
+@cindex bug tracker
 @cindex reporting bugs
 @cindex bugs, reporting
 
-@findex bug_report
-
-If you have Octave working at all, the easiest way to prepare a complete
-bug report is to use the Octave function @code{bug_report}.  When you
-execute this function, Octave will prompt you for a subject and then
-invoke the editor on a file that already contains all the configuration
-information.  When you exit the editor, Octave will mail the bug report
-for you.
-
-If for some reason you cannot use Octave's @code{bug_report} function,
-send bug reports for Octave to @email{bug@@octave.org}.
+To report a bug in Octave, submit a bug report to the Octave bug tracker
+@url{http://bugs.octave.org}.
 
 @strong{Do not send bug reports to @samp{help-octave}}.  Most users of
-Octave do not want to receive bug reports.  Those that do have asked to
-be on the mailing list.
+Octave do not want to receive bug reports.
 
 @node Bug Reporting
 @appendixsec How to Report Bugs
 @cindex bugs, reporting
 
-Send bug reports for Octave to one of the addresses listed in
-@ref{Bug Lists}.
+Submit bug reports for Octave to the Octave bug tracker
+@url{http://bugs.octave.org}.
 
 The fundamental principle of reporting bugs usefully is this:
 @strong{report all the facts}.  If you are not sure whether to state a
 fact or leave it out, state it!
 
 Often people omit facts because they think they know what causes the
 problem and they conclude that some details don't matter.  Thus, you might
 assume that the name of the variable you use in an example does not matter.
@@ -413,12 +392,14 @@ the change, send us the header line.
 The mailing list @email{help@@octave.org} exists for the discussion of
 matters related to using and installing Octave.  If would like to join
 the discussion, please send a short note to
 @email{help@strong{-request}@@octave.org}.
 
 @strong{Please do not} send requests to be added or removed from the
 mailing list, or other administrative trivia to the list itself.
 
-If you think you have found a bug in the installation procedure,
-however, you should send a complete bug report for the problem to
-@email{bug@@octave.org}.  @xref{Bug Reporting}, for
-information that will help you to submit a useful report.
+If you think you have found a bug in Octave or in the installation
+procedure, however, you should submit a complete bug report to
+the Octave bug tracker at @url{http://bugs.octave.org}.
+But before you submit a bug report, please read
+@url{http://www.octave.org/bugs.html} to learn how to submit a useful
+bug report.
diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-03-24  John W. Eaton  <jwe@octave.org>
+
+	* version.h.in (OCTAVE_BUGS_STATEMENT): Point to
+	http://www.octave.org/bugs.html, not the mailing list.
+
 2010-03-24  John W. Eaton  <jwe@octave.org>
 
 	* DLD-FUNCTIONS/config-module.awk: Emit additions to EXTRA_DIST.
 
 2010-03-24  John W. Eaton  <jwe@octave.org>
 
 	* Makefile.am ($(srcdir)/DLD-FUNCTIONS/module.mk):
 	Remove doubled // from file name in rule.
diff --git a/src/version.h.in b/src/version.h.in
--- a/src/version.h.in
+++ b/src/version.h.in
@@ -57,18 +57,17 @@ FITNESS FOR A PARTICULAR PURPOSE." ARG
 #define OCTAVE_WWW_STATEMENT \
   "Additional information about Octave is available at http://www.octave.org."
 
 #define OCTAVE_CONTRIB_STATEMENT \
   "Please contribute if you find this software useful.\n\
 For more information, visit http://www.octave.org/help-wanted.html"
 
 #define OCTAVE_BUGS_STATEMENT \
-  "Report bugs to <bug@octave.org> (but first, please read\n\
-http://www.octave.org/bugs.html to learn how to write a helpful report)."
+  "Read http://www.octave.org/bugs.html to learn how to submit bug reports."
 
 #define OCTAVE_NAME_VERSION_AND_COPYRIGHT \
   OCTAVE_NAME_AND_VERSION "\n" \
   OCTAVE_COPYRIGHT
 
 #define OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_AND_WARRANTY \
   X_OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_AND_WARRANTY("") \
 
